Fix to retrieve oidcEmailAttribute from tokenSet.claims() if not found in standard userInfo

iMarKoLiGa opened the pull request:
Referred to bug [https://github.com/plankanban/planka/issues/884](url)

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/plankanban/planka?pullRequest=885) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/plankanban/planka?pullRequest=885) before we can accept your contribution.<br/><hr/>**Marco Matrella** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/plankanban/planka?pullRequest=885) it.</sub>

meltyshev commented:
I've updated it a bit, added a new environment variable `OIDC_CLAIMS_SOURCE`, which can be set to `OIDC_CLAIMS_SOURCE=id_token` to completely skip fetching userinfo and retrieve all claims from the id_token. Could you please check how this works in your case?

Also, to be able to accept this PR, we need you to sign a CLA. Thank you!

iMarKoLiGa commented:
> I've updated it a bit, added a new environment variable `OIDC_CLAIMS_SOURCE`, which can be set to `OIDC_CLAIMS_SOURCE=id_token` to completely skip fetching userinfo and retrieve all claims from the id_token. Could you please check how this works in your case?
> 
> Also, to be able to accept this PR, we need you to sign a CLA. Thank you!

Sure! I'll make a try and I'll let you know

iMarKoLiGa commented:
@meltyshev I close and re-create this pull request cause of some problems with CLA (i pushed with a wrong username-mail)

meltyshev commented:
> @meltyshev I close and re-create this pull request cause of some problems with CLA (i pushed with a wrong username-mail)

Sure :)
